Assets/MRTK/SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -510,7 +510,7 @@ public virtual void Reset()
 
 #endregion IMixedRealityPointer Implementation
 
-        #region IEquality Implementation
+#region IEquality Implementation
```

Comment:
I think this might be going in the wrong direction. Typically, `#region`s get auto-formatted to align with the code itself:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/350f4cba52b3e16727dcd1c4ca0ed1b0808e0935/Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityInputSystem.cs#L296-L303

Reply:
Fixed!